mlflow/ml-package-versions.yml

```diff
@@ -495,8 +495,9 @@ langchain:
     minimum: "0.0.139"
     maximum: "0.0.143"
     requirements:
-      ">= 0.0.0": ["pyspark", "transformers", "tensorflow", "openai"]
+      ">= 0.0.0": ["pyspark", "transformers", "tensorflow", "openai", "google-search-results"]
```

Comment:
What is this used for?

Reply:
This is required for the langchain agent example which uses `serpapi` as a tool